docs/wiki/development/coding-guidelines.md

```diff
@@ -168,10 +168,18 @@ The family of path macros define global paths to files for use within a module.
 ## 3. Functions
 Functions shall be created in the `functions\` subdirectory, named `fnc_functionName.sqf` They shall then be indexed via the `PREP(functionName)` macro in the `XEH_preInit.sqf` file.
 
-The `PREP` macro allows for CBA function caching, which drastically speeds up load times. **Beware though that function caching is enabled by default and as such to disable it you need to `#define DISABLE_COMPILE_CACHE` above your `#include "script_components.hpp"` include!**
+The `PREP` macro allows for CBA function caching, which drastically speeds up load times. **Beware though that function caching is enabled by default and as such to disable it you need to `#define DISABLE_COMPILE_CACHE` above your `#include "script_component.hpp"` include!**
 
-### 3.1 Headers
-Every function should have a header of the following format as the start of their function file:
+### 3.1 Includes
+Every function includes the `script_component.hpp` file on the first line. Any additional includes or defines must be below this include.
+
+All code written must be below this include and any potential additional includes or defines.
```

Comment:
> additional includes or defines
Duplicated.

Reply:
I thought the same, but think it's just badly worded. Code must appear below all includes and defines.

Includes and defines must appear below the script component include.

Reply:
Hmm, I guess so.

Reply:
This whole document could probably use a once over separately. I just spotted the outdated info while doing this. 👍